### ISSUE TITLE
Add support for add-on icons in EE6 sidebar

### ISSUE THREAD

**Issue by litzinger** [ExpressionEngine/ExpressionEngine#607]
Add-ons have icons, why not use them in the sidebar instead of just displaying the first letter of the add-ons name? Instead of using the default icon, add-ons could have a supplemental icon designed for smaller sizes to be used in the sidebar.

**Comment by dankingsley** [comment]
Agreed! I've currently got 3 addons installed in my EE6 dev site all of which start with the letter S - it makes the collapsed sidebar not really that helpful.

**Comment by intoeetive** [comment]
This is on our to-do list. However the main sidebar navigation is still expecting few other changes to be done, so this might not be implemented in initial 6.0 release
